packages/services/src/services/Translation/__tests__/Translation.test.js

```diff
@@ -76,7 +100,43 @@ describe('TranslationAPI', () => {
         origin: 'https://ibm.com',
       },
     });
-
     expect(response).toEqual(responseSuccess);
   });
+
+  it('should return a json with a recent timestamp', async () => {
+    // using very old cached session
+    sessionStorageMock.setItem(
+      'dds-translation-us-en',
+      JSON.stringify(oldSession)
+    );
+    const previousSession = JSON.parse(
+      sessionStorageMock.getItem('dds-translation-us-en')
+    );
+
+    // reinitializing import
+    const TranslationAPI = (await import('../Translation')).default;
```

Comment:
I won't block proceeding this PR with my comment here, but I'd rather see resetting things done via mocks. The behavior of dynamic import tends to be dependent on build toolchain and we may have high chance to get an undefined behavior if it's used as a mean for "resetting".

Reply:
Ah I see, I'll be applying the same logic for Locale API, so I'll use mocks there and also add it here. Thanks!

Reply:
Any update here...? Thanks!